docs/visibility-on-elasticsearch.md

```diff
@@ -0,0 +1,89 @@
+# Overview
+Cadence visibility APIs allow users to list open or closed workflows with filters such as WorkflowType or WorkflowID.   
+With Cassandra, there are issues around scalability and performance, for example: 
+ - list large amount of workflows may kill Cassandra node.
+ - data is partitioned by domain, which means writing large amount of workflow to one domain will cause Cassandra nodes hotspots.
+ - query with filter is slow for large data.
+
+(With MySQL, there might be similar issues but not tested)
+
+In addition, Cadence want to support users to perform flexible query with multiple filters on even custom info.  
+That's why Cadence add support for enhanced visibility features on top of ElasticSearch (Note as ES below), which includes: 
+ - new visibility APIs to List/Scan/Count workflows with SQL like query
+ - search attributes feature to support users provide custom info
+ 
+# Quick Start
+## Local Cadence Docker Setup
+1. Increase docker memory to higher 6GB. Docker -> Preference -> advanced -> memory limit
+2. Get docker compose file. Run `<code>curl -O [https://raw.githubusercontent.com/uber/cadence/master/docker/docker-compose-es.yml](https://raw.githubusercontent.com/uber/cadence/master/docker/docker-compose-es.yml)</code>`
+3. Start cadence docker which contains Kafka, Zookeeper and ElasticSearch. Run `<code>docker-compose -f docker-compose-es.yml up</code>`
+4. From docker output log, make sure ES and cadence started correctly. If encounter disk space not enough, try `<code>docker system prune -a --volumes</code>`
+5. Register local domain and start using it. `<code>cadence --do samples-domain d re</code>`
+ 
+
+## CLI Search Attributes Support 
+
+Make sure Cadence CLI version is 0.6.4+
+
+### new list API examples
+
+```
+cadence --do samples-domain wf list -q 'WorkflowType = "main.Workflow" and (WorkflowID = "1645a588-4772-4dab-b276-5f9db108b3a8" or RunID = "be66519b-5f09-40cd-b2e8-20e4106244dc")'
+cadence --do samples-domain wf list -q 'WorkflowType = "main.Workflow" StartTime > "2019-06-07T16:46:34-08:00" and CloseTime = missing'
+```
+To list only open workflows, add ``CloseTime = missing`` to the end of query.  
```

Comment:
Do you want backticks around CloseTime = missing?

Reply:
Thanks for catching this. I removed that duplicate backticks.